### ISSUE TITLE
Metrics Browser individual node reporting issue

### ISSUE THREAD

**Issue by jkutes** [Appdynamics/apache-monitoring-extension#2]
Hi,
We have a setup where we are running a different Apache instance on a different VM (virtual machines).
Example:
VM1 running Apache1 (using monitors/ApacheMonitor1)
VM2 running Apache2 (using monitors/ApacheMonitor2)

We are using 4.1.x AppD controller. 

Although the Metric Browser tree for both Apache1 & 2 are present under "Application infrastructure performance->[tier]->Individual Nodes->VM1->Custom metrics", where there should be only one (in this case Apache1, becasue of VM1 is selected).
So there is no data for Apache2 tree metrics there, but the tree is there  which is very clattering/confusing....we are going to have many more different apache instances not only 1 & 2. So this is a problem for us.

Similarly for VM2 there should be only Apache2 instance under individual nodes->VM2, but both trees are present.

So to sum it up: Metrics and their tree on TIER level are reported as expected, but on Individual nodes it's not correct.

I already tried many variation of config.yml to address, but nothing helped.
Did you ever tested setup like this?
Is it possible to address by just modifying the configuration? maybe I have missed something.
If not would this be something you could help us to address by modifying extension code?

Thanks
~juraj

**Comment by jkutes** [comment]
As a workaround I'm using componentID, which is very cumbersome though.
https://docs.appdynamics.com/display/PRO41/Build+a+Monitoring+Extension+Using+Java

Server|Component:18|CustomMetric|Webserver|Apache1

**Comment by akshayAppd** [comment]
The configuration of multiple instances is much simpler now, add as many instances as needed in config.yml under servers with a unique displayName for each
